src/ralph/data_center/models/physical.py

```diff
@@ -176,6 +176,13 @@ class Rack(AdminAbsoluteUrlMixin, NamedMixin.NonUnique, models.Model):
         default=0,
     )
     accessories = models.ManyToManyField(Accessory, through='RackAccessory')
+    require_position = models.BooleanField(
+        default=True,
+        help_text=_(
+            'Uncheck if position is not required for this rack (ex. when rack '
```

Comment:
'Uncheck if position is optional for this rack (ex. when rack '?
This is only a suggestion, do what you wish :)